src/main/java/com/shaneschulte/mc/clanarena/arena/SetupArena.java

```diff
@@ -0,0 +1,225 @@
+package com.shaneschulte.mc.clanarena.arena;
+
+import com.shaneschulte.mc.clanarena.utils.MsgUtils;
+import org.bukkit.Bukkit;
+import org.bukkit.ChatColor;
+import org.bukkit.Location;
+import org.bukkit.boss.BarColor;
+import org.bukkit.boss.BarStyle;
+import org.bukkit.boss.BossBar;
+import org.bukkit.entity.Player;
+
+import java.util.ArrayList;
+import java.util.UUID;
+
+public class SetupArena {
+
+    /**
+     * States for the ArenaSetup
+     */
+    public enum ArenaSetupState {
```

Comment:
why no match consistency with SetupArena

Reply:
changed class to "ArenaSetup"